tailwind.config.mjs

```diff
@@ -15,7 +15,9 @@ export default {
         adderPopDown: 'adderPopDown 0.08s ease-in forwards',
       },
       boxShadow: {
+        DEFAULT: '0 1px 1px rgba(0, 0, 0, 0.1)',
         adderToolbar: '0px 2px 10px 0px rgba(0, 0, 0, 0.25)',
+        sidebar: '0px 1px 4px rgb(0, 0, 0, 0.5)',
```

Comment:
Can you add a comment that this is for the edge of the sidebar container in the host frame.

Reply:
Yep!